app/gui2/ydoc-server/server.ts

```diff
@@ -0,0 +1,7 @@
+import { setupGatewayClient } from './ydoc'
+
+const wss = new WebSocketServer({ host: 'localhost', port: 1234 })
+
+wss.onconnect = (socket, url) => setupGatewayClient(socket, "ws://localhost:8080", url)
```

Comment:
Where is this port and assumed localhost hostname coming from?

Reply:
It will be implemented later. The first goal is to make the server script work.